docs/quick_start/installing.rst

```diff
@@ -59,35 +59,28 @@ Once complete you can add the repository itself into the apt configuration file
     .. code-block:: console
 
        deb  http://archives.fledge-iot.org/latest/bullseye/armv7l/ /
+       deb  http://archives.fledge-iot.org/latest/bullseye/aarch64/ /
 
     to the end of the file.
 
     .. note:: 
-       Replace `bullseye` with  the name of the version of the Raspberry Operating System you have installed.
+       We only offer support for Bullseye on both armv7l and aarch64 architectures, as well as Buster on the armv7l architecture. Use the specific version name of the Raspberry Operating System you have installed.
 
-  - Users with an Intel or AMD system with Ubuntu 18.04 should run
+  - Users with an Intel or AMD system with Ubuntu versions should ensure compatibility with their respective platforms, should run
```

Comment:
better term is _x86_64 or amd64 architectures_ wherever we have written _Intel or AMD system_ or _Intel and AMD architectures_